### PR TITLE
[16.0][MIG] web_m2x_options_manager : Migrated to 16.0.

### PR DESCRIPTION
**Pull request by Ammar-OSI**

### CODE REVIEW

**Comment by yostashiro** [comment]
Thanks for the migration. I might be missing something but the module does not seem to work with a quick functional test in runboat - created a record to remove options from the customer field in sales order, and it does not seem to affect the behavior.

image
image

File: web_m2x_options_manager/__manifest__.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "demo": [
         "demo/res_partner_demo_view.xml",
     ],
-    "depends": ["base", "web_m2x_options"],
+    "depends": ["base"],
```
Comment (anothingguy) [inline web_m2x_options_manager/__manifest__.py]: @Ammar-OSI: `web_m2x_options_manager` should depend on `web_m2x_options`

File: web_m2x_options_manager/static/description/index.html [hunk 2]
```diff
@@ -367,7 +367,7 @@ <h1 class="title">Web M2X Options Manager</h1>
...
+<p><a class="reference external" href="https://odoo-community.org/page/development-status"><img alt="Beta" src="https://img.shields.io/badge/maturity-Beta-yellow.png" /></a> <a class="reference external" href="http://www.gnu.org/licenses/agpl-3.0-standalone.html"><img alt="License: AGPL-3" src="https://img.shields.io/badge/licence-AGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/web/tree/14.0/web_m2x_options_manager"><img alt="OCA/web" src="https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github" /></a> <a class="reference external" href="https://translation.odoo-community.org/projects/web-14-0/web-14-0-web_m2x_options_manager"><img alt="Translate me on Weblate" src="https://img.shields.io/badge/weblate-Translate%20me-F47D42.png" /></a> <a class="reference external" href="https://runbot.odoo-community.org/runbot/162/14.0"><img alt="Try me on Runbot" src="https://img.shields.io/badge/runbot-Try%20me-875A7B.png" /></a></p>
```
Comment (anothingguy) [inline web_m2x_options_manager/static/description/index.html:370]: why we have 14.0 here ? I think it should be 16.0

**Comment by anothingguy** [comment]
@Ammar-OSI  this PR should depend on my PR: https://github.com/OCA/web/pull/2446 so that it can work

**Review by anothingguy — CHANGES_REQUESTED** [review]

**Comment by patrickrwilson** [comment]
@Ammar-OSI can you work on comments and fix pre-commit?

**Comment by XanderDeMatteis** [comment]
@Ammar-OSI this module does also not cover the options "no_create_edit: True" and "no_create": True. When these options are set, the force add command will still not work!

**Comment by QuocDuong1306** [comment]
Hello @Ammar-OSI, thank you for your PR. I have created another PR to take over this PR:
- https://github.com/OCA/web/pull/2587

Your feedback/review is welcome :)

**Pull request closed without merging**